docs/docs/visual-testing-with-storybook.md

```diff
@@ -85,6 +85,9 @@ module.exports = (baseConfig, env, defaultConfig) => {
     require.resolve("@babel/plugin-proposal-class-properties"),
   ]
 
+  // Prefer Gatsby ES6 entrypoint (module) over commonjs (main) entrypoint
+  defaultConfig.resolve.mainFields = ["browser", "module", "main"]
```

Comment:
I'm pretty surprised this isn't the way this already works.

[Webpack works this way](https://webpack.js.org/configuration/resolve/#resolve-mainfields)--and Storybook uses Webpack afaik, so this is _odd_.

Reply:
@DSchau The _defaultConfig_ from storybook has `mainFields: ['browser', 'main', 'module'],`

I think it gets set at https://github.com/storybooks/storybook/blob/0ef3f40d678fb42ca74505e99cc3ad2449c3bfdf/lib/core/src/server/preview/iframe-webpack.config.js#L84

Reply:
I wonder how this will affect other modules outside of gatsby, with that said I think we should  make the change to docs still